main/ui/src/main/resources/fxml/vault_detail_locked.fxml

```diff
@@ -33,7 +33,7 @@
 		<Region VBox.vgrow="ALWAYS"/>
 
 		<HBox alignment="BOTTOM_RIGHT">
-			<Button text="TODO Sanitizer" onAction="#showHealthCheck"/>
+			<Button text="%main.vaultDetail.healthBtn" onAction="#showHealthCheck"/>
```

Comment:
Adding this button in this exact place was just a quick and dirty way to start working on the dialog. I believe there is some better way. @tobihagemann Maybe from within the vault options?

Reply:
Yes, would like to see that button in vault options as well. For now, imo, it'd enough to put it under "General".

Reply:
@infeo Can you move this button to general vault options?

Reply:
Fixed in eb19616